functions.php

```diff
@@ -598,10 +598,19 @@ function edit_searchwp_query_orderby( $orderby ) {
 	public function populate_act_page_children_options() {
 
 		// Get the id of the ACT page. We need this to get the children posts/pages of the ACT Page.
-		$arguments = [
-			'post_type'     => 'page',
-			'post_name__in' => [ 'act', 'ACT', 'Act' ],
-		];
+		$parent_act_id = planet4_get_option( 'select_act_page' );
+		if( 0 != $parent_act_id ) {
```

Comment:
Better check type also here by using !==

Reply:
If we use !== with 0 it check boolean datatypes, because of that I use !=.

Reply:
Yes, Sagar, that is the point. It is in the WordPress coding standards to use strict comparisons.